Fix all default path names to match the ones used on Debian systems.

The spool and overview paths match the ones used by INN 1.x, since TIN is
not able to read the spool of INN 2.x.

--- a/doc/tin.defaults
+++ b/doc/tin.defaults
@@ -27,7 +27,7 @@
 #              files
 # common values are /usr/lib/news, /var/lib/news, /usr/local/lib/news
 #
-#newslibdir=/usr/lib/news
+#newslibdir=/var/lib/news
 
 # bugaddress - defines the email address to which users can send
 #              bugreports using tin's built in function
@@ -46,7 +46,7 @@
 # expects the executable. The program's name is set to inews, don't
 # include it
 #
-#inewsdir=/usr/lib/news
+#inewsdir=/usr/bin
 
 # mm_charset - the default charset to be used in MIME's
 #              Content-Type header
@@ -72,7 +72,7 @@
 # as mail isn't guaranteed to be 8bit clean quoted-printable is
 # a good choice
 #
-#mail_mime_encoding=quoted-printable
+#mail_mime_encoding=8bit
 
 # disable_gnksa_domain_check - allow unregistered top level domains
 #
@@ -88,7 +88,7 @@
 #
 # disable_sender has no effect if inews_prog is not set to --internal.
 #
-#disable_sender=OFF
+disable_sender=ON
 
 ############# reading from a local spool #########################
 #
@@ -107,7 +107,7 @@
 # in a separate directory tree
 # common values are /var/spool/overview, /usr/spool/overview, /news/overview
 #
-#overviewdir=/var/spool/overview
+#overviewdir=/var/spool/news/over.view
 
 # overviewfile - the name of a single overview file
 #
@@ -122,28 +122,28 @@
 # usually the overview.fmt file is in newslibdir, so you only have
 # to change this setting if your configuration differs
 #
-#overviewfmtfile=/usr/lib/news/overview.fmt
+#overviewfmtfile=/etc/news/overview.fmt
 
 # activefile - the full pathname of your newssystem's active file
 #
 # usually the active file is in newslibdir, so you only have
 # to change this setting if your configuration differs
 #
-#activefile=/usr/lib/news/active
+#activefile=/var/lib/news/active
 
 # activetimesfile - the full pathname of your newssystem's active.times file
 #
 # usually the active.times file is in newslibdir, so you only have
 # to change this setting if your configuration differs
 #
-#activetimesfile=/usr/lib/news/active.times
+#activetimesfile=/var/lib/news/active.times
 
 # newsgroupsfile - the full pathname of your newssystem's newsgroups file
 #
 # usually the newsgroups file is in newslibdir, so you only have
 # to change this setting if your configuration differs
 #
-#newsgroupsfile=/usr/lib/news/newsgroups
+#newsgroupsfile=/var/lib/news/newsgroups
 
 # subscriptionsfile - the full pathname of your newssystem's
 #                     subscriptions file
@@ -151,4 +151,4 @@
 # usually the subscriptionsfile file is in newslibdir, so you only have
 # to change this setting if your configuration differs
 #
-#subscriptionsfile=/usr/lib/news/subscriptions
+#subscriptionsfile=/var/lib/news/subscriptions
--- a/src/init.c
+++ b/src/init.c
@@ -686,12 +686,11 @@ init_selfinfo(
 	if (!*subscriptions_file)
 		joinpath(subscriptions_file, sizeof(subscriptions_file), libdir, SUBSCRIPTIONS_FILE);
 	if (!*overviewfmt_file)
-		joinpath(overviewfmt_file, sizeof(overviewfmt_file), libdir, OVERVIEW_FMT);
+		joinpath(overviewfmt_file, sizeof(overviewfmt_file), "/etc/news/", OVERVIEW_FMT);
 	if (!*default_organization) {
 		char buf[LEN], filename[PATH_LEN];
 
-		joinpath(filename, sizeof(filename), libdir, "organization");
-		if ((fp = fopen(filename, "r")) != NULL) {
+		if ((fp = fopen("/etc/news/organization", "r")) != NULL) {
 			if (fgets(buf, (int) sizeof(buf), fp) != NULL) {
 				ptr = strrchr(buf, '\n');
 				if (ptr != NULL)
